src/general/PauseMenu.cs

```diff
@@ -68,6 +87,10 @@ public override void _Ready()
         loadMenu = GetNode<Control>(LoadMenuPath);
         optionsMenu = GetNode<OptionsMenu>(OptionsMenuPath);
         saveMenu = GetNode<NewSaveMenu>(SaveMenuPath);
+        exitConfirmationDialog = GetNode<ConfirmationDialog>(ExitConfirmationDialogPath);
+
+        exitConfirmationDialog.GetOk().Text = TranslationServer.Translate("YES");
+        exitConfirmationDialog.GetCancel().Text = TranslationServer.Translate("NO");
```

Comment:
do this in `ShowExitConfirmationDialog`. Otherwise changing the language ingame breaks this.

Reply:
Good idea, I almost overlooked that.

Reply:
Why isn't there a separate dialog for this? That way godot automatically handles translations. Is there a reason why the confirmation dialog needs to be dynamically have the text changed?

Reply:
The ConfirmationDialog node is already suitable for this purpose but the button texts doesn't quite match given the context so it just needs a little bit tweaking to them. I guess a WindowDialog node can be used but we would have to basically readd some parts for the same functionality.

Reply:
I guess I'll be fine with the less duplication and problems creating alternative.